docker-compose.yml

```diff
@@ -13,7 +13,7 @@ services:
 
   # Hasura
   graphql-engine:
-    image: hasura/graphql-engine:v2.13.2.cli-migrations-v3
+    image: hasura/graphql-engine:v2.19.0.cli-migrations-v3
```

Comment:
Our cloud is running on v2.18.0-cloud.2, so I propose we upgrade to that, but not beyond it for now.
<img width="359" alt="Screenshot 2023-02-20 at 11 03 44 AM" src="https://user-images.githubusercontent.com/83605543/220182578-f82256c5-7bc4-47e4-8fb8-d80b4ab52395.png">

Reply:
i've pushed a commit with 2.18